### PR TITLE
https://github.com/search?utf8=✓&q=extension%3Arexx+NOT+nothack&type=Code&ref=searchresults

### PR DESCRIPTION
**Pull request by idrougge**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -707,3 +707,9 @@
 [submodule "vendor/grammars/atomic-dreams"]
 	path = vendor/grammars/atomic-dreams
 	url = https://github.com/PJB3005/atomic-dreams
+[submodule "rexx-sublime"]
```
Comment (larsbrinkhoff) [inline .gitmodules]: This doesn't look right.

**Comment by larsbrinkhoff** [comment]
Hello @idrougge, and thank you for your contribution!

I see your search have 2000 matches, and REXX certainly has a long history.  I'm surprised on one added it yet!

However, some remarks:
- You should note the origin and license of the samples.
- I didn't find many REXX files using the `.rx` extension.  Actually, more are Forth!
- You're touching (almost?) all grammar files.

**Comment by idrougge** [comment]
@larsbrinkhoff 
• Origin of the sample files is me, and I forfeit any licence to those.
• I can remove .rx extension.
• I think grammar files were touched by script/licensed. How do I tell it not to?

**Comment by larsbrinkhoff** [comment]
Regarding the last point, try

    git submodule deinit -f .
    git submodule update --init

**Comment by idrougge** [comment]
@larsbrinkhoff Afterwards, do I run the scripts/* again?

**Comment by larsbrinkhoff** [comment]
Sorry, I don't know.  I haven't run into this problem myself.  I usually run those scripts rarely.

**Comment by idrougge** [comment]
@larsbrinkhoff Pushed update.

**Comment by larsbrinkhoff** [comment]
Sorry, that didn't work.  Maybe try the instructions in [this comment](https://github.com/github/linguist/pull/3008#issuecomment-230461471)?

**Comment by larsbrinkhoff** [comment]
I made a pull request for your pull request. :-)

**Comment by larsbrinkhoff** [comment]
Great!  This looks much better.

Now, I suppose the GitHub master branch has moved forward since you started your pull request.  You should do a merge or rebase and resolve the conflicts.

Sorry for all the trouble, it's usually much smoother than this.

By the way, I see your samples has Swedish text in them.  Så, hälsningar från en annan svensk!

**Comment by larsbrinkhoff** [comment]
Hello,

There are still some problems.  I took the liberty of making a new pull request with your changes: #3124.

**Comment by larsbrinkhoff** [comment]
@idrougge Two questions:
- Does my reworked pull request seem ok to you?
- Why are there DEL (hex encoding FF, unicode ÿ) characters in the BatchRemapBrushes.pprx sample?  E.g.  
`PARSE VAR fillista filnamn 'ÿ' fillista`

**Comment by idrougge** [comment]
@larsbrinkhoff 
* Yes, but I've just found out that some ARexx scripts can be named .wb
* $FF is an unlikely part of a file name, so I used it as a separator in a file list. Others might use CR for the same purpose.

BTW, I thought I recognised your name – I know NoCrew from the Atari scene.

**Comment by larsbrinkhoff** [comment]
I believe this is fixed now, right?  If not, please reopen.

**Pull request closed without merging**